BrainPortal/app/helpers/select_box_helper.rb

```diff
@@ -613,5 +613,14 @@ def uploadable_data_providers(user=current_user)
     return uploadable_data_providers
   end
 
+  # returns currently active project if user can assign to it or own group id. Serves as default destination project
```

Comment:
I don't know why this method is in this file instead of `session_helper.rb`; move it there

Also, make one invoke the other so that you do not duplcaite code; e.g. the _group method has the logic and the _group_id method invokes the group method and then calls `.id`

Reply:
Somehow it did not work from template, I can try again but
@prioux  what about dropping the whole tag template commit? That code ('default_project') is dead and affect nothing.

Reply:
I don't understand what you are talking about. What tag template? What dead code? Can you please explain?

Reply:
the method `current_assignable_group` is used only to set variable `default_project`, the rest of code use `current_assignable_group_id`. While that variable is nominally used in the template `dialogs.html.erb` , however it does affect the behavior ( tag is defaulted to the first assignable workgroup ). So I suggested to remove the changes related to that template and method. That is revert commit

https://github.com/MontrealSergiy/cbrain/commit/6873f053818cad6a3a9ca89ffa9947e3be48a27a

Reply:
Sure, revert that commit if you want.

Reply:
I still don't know which 'dead code' you were referring to... I don't see any dead code.

Reply:
You are right, then, my bad. And I am doing as you suggested. 

But probably `current_assignable_group_id` can be replaced with `current_assignable_group.id`.

Alternatively commit still can be dropped because non-assignable projects seem to be filtered somewhere else and tag is not as offensive as a new file.

Reply:
Yes, it might be worth it to have only the method `current_assignable_group` and then do `current_assignable_group.id` when we need the ID.

You can make that change if you want, everywhere you invoke the _id method right now (just make sure not to forget any place)

Reply:
done